plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1608,7 +1594,7 @@ read_only::get_table_by_scope_result read_only::get_table_by_scope( const read_o
                                                                     const fc::time_point& deadline )const {
 
    fc::microseconds params_time_limit = p.time_limit_ms ? fc::milliseconds(*p.time_limit_ms) : fc::milliseconds(10);
```

Comment:
```suggestion
   fc::microseconds params_time_limit = p.time_limit_ms ? fc::milliseconds(*p.time_limit_ms) : fc::microseconds::maximum();
```

Reply:
The default if not provide by the user is still 10ms I thought. Our proposal I don't think changed that.

Reply:
Yes, that's true, we didn't specify that in the proposal,. I think it would be more logical to say that if the user doesn't specify an explicit time limit, the `http_max_response_time` would then be the limit, rather than having a additional hardcoded 10ms here that we didn't explicitly document. But it is not a major issue imo and we can always fine tune that later if we want.

Reply:
I'm not opposed to making that change. Seems like now is the time to do it along with all the other changes. Should we also reduce the default `http-max-response-time-ms` to say 15ms from 30ms.

Reply:
Sounds good to me (reducing to 15ms). This is a value that can easily be changed by the node operator if necessary, and we get rid of this hard-coded 10ms which I think doesn't bring much to the table.

Reply:
There are a few spots with this 10ms setting. I like it when we make things simpler :-).

Reply:
I agree. I'll add it to the PR description. Seems like the logical way to go.